File: week-07/day-01/draw-pyramid.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

var stars = '';
var spaces = '';
var limit = 0;
for (let i = 0; i < lineCount; i++) {
  for (let j = lineCount - 1; j > i; j--) {
    spaces += " ";
  }
  for (let j = 0; j <= limit; j++ ) {
    stars += "*";
  }
  limit += 2;
  console.log(spaces + stars);
  stars = '';
  spaces = '';
}

File: week-02/day-01/draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was
# lines = int(input("Number of lines: "))
lines = 5
stars = ""
spaces = ""
j = 1
for i in range(0, lines):
    for k in range(0, j):
        stars += "*"     
    for k in range(0, lines - i):
        spaces += " "
    print (spaces + stars + "\n")
    stars = ""
    spaces = ""
    j += 2